crates/trippy-tui/src/frontend/columns.rs

```diff
@@ -271,8 +271,11 @@ impl ColumnType {
     /// The name of the column in the current locale.
     #[allow(clippy::cognitive_complexity)]
     pub(self) fn name(&self) -> Cow<'_, str> {
+        if self == &Self::Ttl {
+            return "#".into();
+        }
         match self {
-            Self::Ttl => Cow::Borrowed("#"),
+            Self::Ttl => unreachable!(),
```

Comment:
Avoids introducing `.into()` to all `t!()` invocations at the cost of extra statements other than the match. I'll switch if you'd prefer.

Reply:
This got me thinking:

The `ColumnType::name` method here returns `Cow<'_, str>` only because that is what the `i18n-rust` crate macro returned.  Easy to fix, we just make the return type here (and a few other places) a String.  

However, does that mean that `i18n-embed` allocates an owned string every time? After digging through the code of the `i18n_embed_fl::fl!` macro it seems it ultimate calls this:

https://github.com/kellpossible/cargo-i18n/blob/master/i18n-embed/src/fluent.rs#L133-L136

```rust
/// Get a localized message referenced by the `message_id`.
pub fn get(&self, message_id: &str) -> String {
    self.get_args_fluent(message_id, None)
}
```

So the answer appears to be that it does allocate a new String for every translation, every time.

Reply:
`i18n_embed::fluent::FluentLanguageLoader::get_args_fluent()` ultimately calls [`fluient::bundle::FluentBundle::format_pattern()`](https://docs.rs/fluent/latest/fluent/bundle/struct.FluentBundle.html#method.format_pattern), which actually returns a `Cow<_, str>`. It then turns it `.into()` a `String`. This is a bit frustrating.

Reply:
I ran a similar test of trippy 0.12.0 vs this branch (build in release mode) and there is no significant difference in CPU usage on my system (`x86_64` MacBook Pro).  I also compared vs 0.11.0. The numbers are all within the margin of error.

0.11.0: ~1.5%
0.12.0: ~1.6%
0.13.0-dev: ~1.7%

So I don't think this is a real-world concern.